### ISSUE TITLE
monospace flag not set

### ISSUE THREAD

**Issue by nocantsin** [iaolo/iA-Fonts#73]
The font isn't defined/flagged as a monospace font in its meta data, which makes it impossible to use it (among others) as a terminal font in Linux.

**Comment by Raccffy** [comment]
That's true - Windows Terminal does not display iA Writer Mono font either (although it was installed properly). Here's the [fixed version](https://github.com/Raccffy/iA-Fonts).
